proto/rill/runtime/v1/resources.proto

```diff
@@ -111,10 +111,10 @@ message ModelSpec {
   bool incremental = 13;
   string incremental_state_resolver = 14;
   google.protobuf.Struct incremental_state_resolver_properties = 15;
-  string splits_resolver = 18;
```

Comment:
Do we need to keep this for backwards compatibility ?

Reply:
This change is actually backwards compatible because the renamed field uses the same number (`18`). Protobuf stores fields with these numbers, not their names. So renaming only changes the names in JSON serialization in the APIs, but we don't need backwards compatibility there.